app/src/main/res/values/strings.xml

```diff
@@ -357,6 +357,7 @@
   <string name="statistics">STATISTICS</string>
   <string name="statistics_thanks">Thanks Received</string>
   <string name="statistics_featured">Featured Images</string>
+  <string name="statistics_wikidata_edits">Wikidata edits through app</string>
```

Comment:
I just noticed that other strings have all of their words' first letter uppercase. I am not sure about the convention, but better make everything use the same convention. In my opinion only the first word's first letter should be uppercase.

Reply:
so changing the others is an option.

Reply:
Sure will do. Also, does this string(Wikidata edits through app) convey the meaning properly or can we have something better?

Reply:
Indeed, some people might use Nearby without having ever heard about "Wikidata". Even Nearby's tooltip does not talk about Wikidata.

The wording is difficult, though.
How about `Images that were needed to illustrate articles` or `Images via "Nearby Places"`?

Reply:
Am changing it to `Images via "Nearby Places"`

Reply:
To maintain consistentcy, I am thinking of keeping strings in title case itself.

Reply:
I vote for only making the first word's first letter uppercase (so, changing the other strings)